selene_sdk/samplers/multi_file_sampler.py

```diff
@@ -3,45 +3,56 @@
 FileSampler for each mode of sampling (train, test, validation).
 The MultiFileSampler is therefore a subclass of Sampler.
 """
+import numpy as np
+from torch.utils.data import DataLoader
 
 from .sampler import Sampler
 
 
 class MultiFileSampler(Sampler):
```

Comment:
Wondering if we should actually deprecate `MultiFileSampler` and just call this `MultiSampler` or something? We can keep `MultiFileSampler` and update the documentation / output a usage warning to say that new versions of Selene no longer maintain that class and instead recommend `MultiSampler`

Reply:
That's a good idea!